### ISSUE TITLE
[css-images] 1D "painting area" for stripes() is confusing

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#8160]
`stripes()` keeps referring to a "painting area", e.g.

> a [`<percentage>`](https://drafts.csswg.org/css-values-4/#percentage-value) is relative to the painting area

But this is very confusing since an area is 2D but `stripes()` is a `<1d-image>`. And for example,

```css
width: 0;
height: 0;
border: 50px solid;
border-radius: 100%;
border-color: stripes(blue 50%, orange 50%);
```

I think it would look like the image below, so the actual areas are in a proportion of 1:4, not 1:1
![](https://user-images.githubusercontent.com/7477678/204861053-c41a0430-690e-48ac-af2d-1a8290c3106c.png)

I think "painting area" should be renamed to something like "length of the paint line".

**Comment by SebastianZ** [comment]
I agree that "painting area" sounds confusing, though @fantasai already changed that term to "total width". So I guess this issue can already be closed.

Sebastian

**Comment by Loirooriol** [comment]
a71367ac45bd15fbb1727ce7691bb7abee6e0010 That's good, but I'm now it seems that each consumer of `<image-1D>` should define both the paint line and the total width. Maybe just define the total width as the length of the paint line?

**Comment by SebastianZ** [comment]
> Maybe just define the total width as the length of the paint line?

I did just that now in #10777.

Sebastian
